tests/unit/test_swarm_manager.py

```
@@ -1,6 +1,7 @@
 """
 Unit tests for spds.swarm_manager module.
 """
+import json
 import pytest
 from unittest.mock import Mock, patch, MagicMock, call
```

Flake8 (7.2.0)

[error] 'unittest.mock.MagicMock' imported but unused

(F401)